frontends/PyCDE/src/pycde/system.py

```diff
@@ -174,12 +173,13 @@ def print(self, *argv, **kwargs):
 
   def graph(self, short_names=True):
     import mlir.all_passes_registration
-    pm = mlir.passmanager.PassManager.parse("view-op-graph{short-names=" +
-                                            ("1" if short_names else "0") + "}")
+    pm = mlir.passmanager.PassManager.parse(
```

Comment:
These python changes LGTM but FWIW (cc @mikeurbach) we can 1)construct a passmanager with the anchor op as argument, if that's known/helps and 2)can incrementally build up passmanager's with new `add(..)` method instead of putting together textual pipeline explicitly first, see: https://reviews.llvm.org/D137344 .

As for handling LLVM bump I think this is the way to go, but FYI in case that helps re:python bits in the future.

Reply:
That's neat, thanks for pointing it out. Agreed that for LLVM bump this mechanical change is the way to go.

Reply:
I agree with Mike. Thanks for updating pycde as well!